docs/guides/mssqlserver/concepts/mssqlserver.md

```diff
@@ -0,0 +1,482 @@
+  - lifecycle
+
+You can check out the full list [here](https://github.com/kmodules/offshoot-api/blob/master/api/v2/types.go#L26C1-L279C1).
+Uses of some field of `spec.podTemplate` is described below,
+
+#### spec.podTemplate.spec.tolerations
+
+The `spec.podTemplate.spec.tolerations` is an optional field. This can be used to specify the pod's tolerations.
+
+#### spec.podTemplate.spec.volumes
+
+The `spec.podTemplate.spec.volumes` is an optional field. This can be used to provide the list of volumes that can be mounted by containers belonging to the pod.
+
+#### spec.podTemplate.spec.podPlacementPolicy
+
+`spec.podTemplate.spec.podPlacementPolicy` is an optional field. This can be used to provide the reference of the podPlacementPolicy. This will be used by our Petset controller to place the db pods throughout the region, zone & nodes according to the policy. It utilizes kubernetes affinity & podTopologySpreadContraints feature to do so.
+
+
+#### spec.podTemplate.spec.containers
+
+The `spec.podTemplate.spec.containers` can be used to provide the list of containers and their configurations for to the database pod. some of the fields are described below,
+
+##### spec.podTemplate.spec.containers[].name
+The `spec.podTemplate.spec.containers[].name` field used to specify the name of the container specified as a DNS_LABEL. Each container in a pod must have a unique name (DNS_LABEL). Cannot be updated.
+
+##### spec.podTemplate.spec.containers[].args
+`spec.podTemplate.spec.containers[].args` is an optional field. This can be used to provide additional arguments to database installation.
+
+##### spec.podTemplate.spec.containers[].env
+
+`spec.podTemplate.spec.containers[].env` is an optional field that specifies the environment variables to pass to the MSSQLServer docker image. To know about supported environment variables, please visit [here](https://hub.docker.com/r/microsoft/mssql-server).
+
+Note that, the KubeDB operator does not allow `MSSQL_SA_USERNAME` and `MSSQL_SA_PASSWORD` environment variable to set in `spec.podTemplate.spec.env`. If you want to set the superuser _username_ and _password_, please use `spec.authSecret` instead described earlier.
+
+If you try to set `MSSQL_SA_USERNAME` or `MSSQL_SA_PASSWORD` environment variable in MSSQLServer CR, KubeDB operator will reject the request with following error,
+
+```ini
+The MSSQLServer "mssqlserver" is invalid: spec.podTemplate: Invalid value: "mssqlserver": environment variable MSSQL_SA_PASSWORD is forbidden to use in MSSQLServer spec
+```
+
+Also, note that KubeDB does not allow to update the environment variables as updating them does not have any effect once the database is created.
+
+##### spec.podTemplate.spec.containers[].resources
+
+`spec.podTemplate.spec.containers[].resources` is an optional field. This can be used to request compute resources required by containers of the database pods. To learn more, visit [here](http://kubernetes.io/docs/user-guide/compute-resources/).
```

Comment:
`[here](http://kubernetes.io/docs/user-guide/compute-resources/).`
this gives 404 error.